saleor/webhook/transport/tests/test_response_schemas.py

```diff
@@ -0,0 +1,352 @@
+import base64
+from decimal import Decimal
+from unittest.mock import patch
+
+import graphene
+import pytest
+from pydantic import ValidationError
+
+from ...response_schemas import (
+    ExcludedShippingMethodSchema,
+    FilterShippingMethodsSchema,
+    ListShippingMethodsSchema,
+    ShippingMethodSchema,
+    logger,
+)
+
+
+def decode_id(id):
+    return base64.b64decode(id).decode()
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        # All fields provided
+        {
+            "id": "1",
+            "name": "Standard Shipping",
+            "amount": Decimal("10.00"),
```

Comment:
Maybe we should extend the tests with input data of different types for `amount: Decimal = Field(ge=0)` field? `int`, `float` and `str` should be supported